pipeline_steps/artifact_build.groovy

```diff
@@ -108,4 +108,30 @@ def container(String image) {
   } // pubcloud slave
 }
 
+def cloudimage(Map args) {
+  inventory="inventory.${common.rand_int_str()}"
+  inventory_path="${WORKSPACE}/rpc-gating/playbooks/${inventory}"
```

Comment:
This should be passed into getPubCloudSlave

Reply:
Done.

Reply:
Sorry, I should have been more explicit, inventory_path is required as well.

Reply:
Ah, thanks for clarifying via IRC. Done.